### PR TITLE
created vote.css

### PR DESCRIPTION
**Pull request by Shreya-L**

### CODE REVIEW

File: animations/vote.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  50%  {background-color: white;}
+  100% {background-color: blue;}
+}
+
+.vote {
+	font-family: 'Helvetica Neue', 'Helvetica';
+	text-transform: uppercase;
+	color: #ff000;
+	text-align: center;
+	animation: vote;
```
Comment (NiallEccles) [inline animations/vote.css:21]: Just need to add in a time after `vote`! At the minute it doesn't play so you could just set it to be `vote 1s`

File: animations/vote.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  
+  0%   {background-color: red;}
+  50%  {background-color: white;}
+  100% {background-color: blue;}
+}
+
+.vote {
+	font-family: 'Helvetica Neue', 'Helvetica';
+	text-transform: uppercase;
+	color: #ff000;
```
Comment (NiallEccles) [inline animations/vote.css:19]: I think you are missing another 0 here. `#ff000` -> `#ff0000`

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Hey there 👋 Thanks for submitting this pull request! I just have two really small changes for you to make before I can merge this.

**Comment by Shreya-L** [comment]
Thanks, updated both of those!

**Comment by NiallEccles** [comment]
Excellent!

**Pull request merged**